### ISSUE TITLE
About Generative module and GAN loss 

### ISSUE THREAD

**Issue by YangJae96** [chenhao2345/GCL#10]
![image](https://user-images.githubusercontent.com/57695621/142572119-06111179-0af9-4885-85e8-f4bc9c1a746d.png)
The above is the generative module in the paper. 

Can I ask why you applied the x'org part in the GAN loss?

Is it because the training generator is hard without using x'org part?

**Comment by chenhao2345** [comment]
Yes, it helps to train generator. This part is traditional reconstruction in GAN and AE. Reconstruction allows for better adapting the decoder to the person distribution.
